node/src/neighborhood/mod.rs

```diff
@@ -2836,6 +2867,7 @@ mod tests {
     ) {
         let system = System::new("route_query_responds_with_none_when_asked_for_one_hop_round_trip_route_without_consuming_wallet_when_back_route_needs_two_hops");
         let mut subject = make_standard_subject();
+        subject.min_hops_count = Hops::OneHop;
```

Comment:
This is a bad test. Not your fault, but it claims to be stimulating a failed query because of a hop-count problem when the query is actually failing because there are no full neighborships.

Recommendation: briefly change `.add_arbitrary_half_neighbor` below to `.add_arbitrary_full_neighbor`. Run the test. If it passes, leave it that way. If it fails, put it back the way it was and make a card about cleaning it up.

Reply:
I've created this card - https://github.com/MASQ-Project/MASQ-Node-issues/issues/704